CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 6.1.0
+
+* Add "Register to vote" section to related links data. This will only affect
+  completed transactions (apart from register-to-vote) until May 23rd.
```

Comment:
Given that this `CHANGELOG` might be around for a while, it might be best to specify the year this should run to?